documentation/markup.rst

```diff
@@ -229,13 +258,18 @@ body at the bottom of the document after a "Footnotes" rubric heading, like so::
 
 You can also explicitly number the footnotes for better context.
 
+.. _comments:
 
 Comments
 --------
 
 Every explicit markup block which isn't a valid markup construct (like the
 footnotes above) is regarded as a comment.
 
+Comments are generally preceeded by ``..``::
```

Comment:
```suggestion
Every explicit markup block (starting with ``.. ``)
which isn't a :ref:`valid markup construct <directives>`
is regarded as a comment::
```

* Add the mention of "starting with `..`" up here instead
* Avoid double paren and relying on the specific structure of the text
* Link the directive entry explaining the valid markup constructs (and explicit markup blocks)
* Simply and avoid redundancy
* Fix typo ("preceeded")

Since `..` _is_ the start of an explicit markup block as stated in the previous line, comments *must* be therefore be preceded by such, not just most of the time. Since this is therefore mostly redundant, I just combined the two lines